trace/trace_opentelemetry/main.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Google LLC
```

Comment:
File name comment: we can drop `trace_` in `trace_opentelemetry` because we are in the `trace` directory.

If OpenTelemetry samples will have more than just tracing, it may make more sense to create a top-level `opentelemetry` directory for those samples. We usually have one top-level directory per product.

Reply:
Wouldn't that be `monitoring/opentelemetry` and so on to keep the existing directory taxonomy?